lib/triBoxOverlap.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+/**
+ * Function and helpers for computing the intersection between a static triangle and a static AABB.
+ * Adapted from Tomas Akenine-Möller's public domain implementation: http://www.cs.lth.se/home/Tomas_Akenine_Moller/code/
```

Comment:
Looks like the third party code section is pretty out of date, should I open an issue for that or just update it here?